src/styles/main.scss

```
@@ -1,3 +1,238 @@
+@font-face {
+  font-family: "Roboto";
+  src: url(/fonts/Roboto-Regular.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Roboto-Medium";
+  src: url(/fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "Roboto-Bold";
```

```suggestion
  font-family: "Roboto";
  src: url(/fonts/Roboto-Medium.ttf);
  font-style: normal;
  font-weight: 500;
}

@font-face {
  font-family: "Roboto";
```
font-family should have same name